main.js

```diff
@@ -0,0 +1,195 @@
+
+const myArr = [
+  new SpeakerData(
+    1,
+    'Satoshi Nakamoto',
+    'image/satoshi-nakamoto-profile.jpg',
+    'Software Developer',
+    'In his career as Dev he create, desing and develop the first decentralize protocol for value exchange',
+  ),
+  new SpeakerData(
+    2,
+    'Vitalik Buterin',
+    'image/vitalik-buterin-profile.jpg',
+    'Software Developer',
+    'He take part as a blockchain Dev group and desing the Ethereum protocol. Wicht integrate the smart contract to a block-chain network',
+  ),
+  new SpeakerData(
+    3,
+    'Gavin Wood',
+    'image/gavin-wood.jpg',
+    'Polkadot CEO',
+    'He take part as a blockchain Dev group and desing the Polkadot protocol. Wicht integrate a second layer to Ethereum network',
+  ),
+  new SpeakerData(
+    4,
+    'Changpeng Zhao',
+    'image/changpeng-zhao.jpg',
+    'Binance CEO',
+    'In his career he create important project like Binance Exchange & Binance Smart Chain.',
+  ),
+  new SpeakerData(
+    5,
+    'Michael Saylor',
+    'image/michael-saylor.jpg',
+    'Microstrategy CEO',
+    'Big supporter of block-chain tecnology, his company is one of the bigger BTC holder',
+  ),
+  new SpeakerData(
+    6,
+    'Charles Hoskinson',
+    'image/charles-hoskinson.jpg',
+    'Software Developer',
+    'He take part as a blockchain Dev group and desing the Cardano protocol',
+  ),
+];
+
+// define function to use my data
+
+function creatSpeakerSection(i) {
+  // create div element for speaker sections
+
+  const speakerDiv = document.createElement('div');
+  speakerPage.appendChild(speakerDiv);
+  speakerDiv.classList.add('speaker-page-container');
+  if (i > 1) {
+    speakerDiv.classList.add('show-less');
+  }
+
+  // create img element for speaker sections
+
+  const speakerImg = document.createElement('img');
+  speakerDiv.appendChild(speakerImg);
+  speakerImg.classList.add('speaker-page-container-img');
+  speakerImg.src = myArr[i].imgProfile;
+
+  // create div element for headers
+
+  const speakerDiv2 = document.createElement('div');
+  speakerDiv.appendChild(speakerDiv2);
+  speakerDiv2.classList.add('speaker-page-container-2');
+
+  // create h1 element for speaker sections
+
+  const speakerH1 = document.createElement('h1');
+  speakerDiv2.appendChild(speakerH1);
+  speakerH1.classList.add('speaker-page-container-h1');
+  speakerH1.textContent = myArr[i].name;
+
+  // create h2 element for speaker sections
+
+  const speakerH21 = document.createElement('h2');
+  speakerDiv2.appendChild(speakerH21);
+  speakerH21.classList.add('speaker-page-container-h2-1');
+  speakerH21.textContent = myArr[i].workTitle;
+
+  // create h2 element for speaker sections
+
+  const speakerH22 = document.createElement('h2');
+  speakerDiv2.appendChild(speakerH22);
+  speakerH22.classList.add('speaker-page-container-h2-2');
+  speakerH22.textContent = myArr[i].workDescription;
+}
+
+for (let i = 0; i <= 5; i += 1) {
+  creatSpeakerSection(i);
+}
```

Comment:
- [ ] kindly consider sticking to their provided design as much as possible to meet the below requirements
<img width="1042" alt="Screen Shot 2023-05-25 at 07 15 17" src="https://github.com/Zven94/conference-page/assets/60146030/be1d6557-35ab-4e6f-a365-bdd43501d2ee">
 check the images below and see the difference and consider fixing your design before you proceed
- your design 👇 
<img width="762" alt="Screen Shot 2023-05-25 at 07 17 59" src="https://github.com/Zven94/conference-page/assets/60146030/371da53b-9bfc-4656-8ce5-3206bfee4ff2">


- check everything I pointed below in the  given design and fix it👇 
<img width="929" alt="Screen Shot 2023-05-25 at 07 19 16" src="https://github.com/Zven94/conference-page/assets/60146030/9e574787-e86b-49f2-8184-750be67c8fce">